public/app/features/inspector/InspectJSONTab.tsx

```diff
@@ -121,6 +122,14 @@ export class InspectJSONTab extends PureComponent<Props, State> {
         } else {
           const updates = JSON.parse(this.state.text);
           dashboard!.shouldUpdateDashboardPanelFromJSON(updates, panel!);
+
+          //Report relevant updates
+          reportInteraction('grafana_panel_inspect_applied_clicked', {
```

Comment:
Should it be "apply clicked"? 🤔 
```suggestion
          reportInteraction('grafana_panel_inspect_apply_clicked', {
```